next/components/forms/steps/Summary/FormSummary.tsx

```diff
@@ -20,7 +20,7 @@ const FormSummary = () => {
           // The validator is not used, but it's required by the form. We use our own validation in `useFormSummary`.
           validator={rjfsValidator}
           experimental_defaultFormStateBehavior={defaultFormStateBehavior}
-          readonly
+          readonly={isReadonly}
```

Comment:
Summary is always readonly, as it cannot change the data in any way. Please revert this.